### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by Nelitaa**
-Add "Works" section (use CSS Grid and Flexbox combined)
-Add "About myself" section (with flexbox)
-Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Nelitaa, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Meaningful commit messages ✔️ 
- All linter passed 👍🏼 
- Readme is professional ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -19,13 +20,96 @@
...
+                    <ul class="lenguages">
+                        <li class="boxlenguages">Html</li>
+                        <li class="boxlenguages">Css</li>
+                        <li class="boxlenguages">Javascript</li>
+                    </ul>
+                    <div class="button" >
+                        <button class="button2" type="submit">See this project</button>
+                    </div>
+                </div>
+            </div>
```
[the remark this answers is not in the document]
Comment (Nelitaa) [reply index.html:47]: Done! Thanks for the review!

File: index.html [hunk 2]
```diff
@@ -19,13 +20,96 @@
...
-            <div class="button1" >
-                <button type="submit">Start collaboration</button>
+            <div class="button" >
+                <button class="button1" type="submit">Start collaboration</button>
             </div>
         </section>
         <section class="content2">
             <div class="imgback1"><br></div>
         </section>
+        <section class="works">
```
[the remark this answers is not in the document]
Comment (mgmediaweb) [reply index.html:35]: Good job following the advice of the previous reviewer

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (mgmediaweb) [reply index.html]: Good job following the advice of the previous reviewer

**Review by mgmediaweb — APPROVED** [review]
Hi @Nelitaa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved5

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**